### ISSUE TITLE
DICe for rotate measurement

### ISSUE THREAD

**Issue by shang1117** [dicengine/dice#321]
Dice plays a very important role in the development of our project. When we recently conducted binocular DIC measurements of rotating disks, we found that the calculated results had a significant deviation. I guess there may be a problem with the matching between the rotation angle of the deformed image and the reference image, and it may be necessary to provide an initial value such as an angle before matching. Could you please confirm my speculation and whether DICe is not suitable for 3D measurement of this type of rotating specimen?

**Comment by dicengine** [comment]
If you have large rotations between frames, you may have to use one of the initialization methods that's not readily available in the GUI options. If you are able to post a couple of example images from two frames that might help determine which initialization method is best. DICe has been used for rotating rotor blades so it's probably a matter of using the right initialization method for the correlation between frames.

**Comment by shang1117** [comment]
yes please. Here we post a couple of example images from two frames. thanks!
![L-17](https://github.com/dicengine/dice/assets/103749437/6f06f97d-eb7f-41a7-b441-974b03d06717)
![R-0](https://github.com/dicengine/dice/assets/103749437/0afaa074-a535-461a-a6a2-ebe5b79f7ff7)
![R-17](https://github.com/dicengine/dice/assets/103749437/26a3557b-f9b4-4593-8a72-8dd6282df16d)

![L-0](https://github.com/dicengine/dice/assets/103749437/5704f7b4-85ed-4831-9b7c-2c2900c6d014)

**Comment by dicengine** [comment]
Take a look at this regression test: https://github.com/dicengine/dice/tree/master/tests/regression/image_reg_init
In particular, see how the params.xml options are set. I think if you just change the initialization type to USE_IMAGE_REGISTRATION it might work. You will have to run DICe from the command line (not from the GUI because clicking the run button in the GUI will override your changes to the params.xml file).
